src/frontend/src/pages/login/login.tsx

```diff
@@ -4,4 +4,5 @@ export const Login: React.FC = () => {
     return (
         <div>Login Page</div>
     )
-};
+    
+};
```

Comment:
```suggestion
};

```
This should get rid of this diff.